canonical/cloud-init
# Fix cc_keyboard in mantic

## Proposed Commit Message


```
Fix cc_keyboard in mantic

localectl set-x11-keymap is currently broken for new debian based
installs. Workaround it by writing /etc/default/keyboard directly

LP: #2030788
```

## Additional Context
https://bugs.launchpad.net/ubuntu/+source/systemd/+bug/2030788
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1038762
`man 5 keyboard`

## Test Steps
I haven't been able to verify this manually, so it may be worth asking the submitter to verify first
